### PR TITLE
Add QueryByExample docs

### PR DESCRIPTION
**Pull request by BigBlueHat**
This PR includes a wee bit of reorg and some code block colorizing.

### CODE REVIEW

File: developers/issuers.md [hunk 1]
```diff
@@ -28,13 +28,13 @@ await credentialHandlerPolyfill.loadOnce();
...
-```
+```sh
 npm i credential-handler-polyfill@3.0.1
-``` 
+```
 
 and then import and load the polyfill library as follows:
 
-```
+```javascript
```
Comment (davidlehn) [inline developers/issuers.md:37]: Could use "js" here and elsewhere.

**Review by davidlehn — APPROVED** [review]

File: developers/verifiers/index.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
 ---
 layout: subpage
 title: "Developer Docs for VC Verifiers"
-permalink: /developers/verifiers
+permalink: /developers/verifiers/
 ---
 ## CHAPI for VC Verifiers
-CHAPI integrates easily into verifier websites, allowing your site to request Verifiable Credentials presentations from a user's digital wallet
+CHAPI integrates easily into verifier websites, allowing your site to request Verifiable Credentials presentations from a user's digital wallet.
```
Comment (msporny) [inline developers/verifiers/index.md:7]:
```suggestion
CHAPI integrates easily into verifier websites, allowing your site to request Verifiable Credentials from a digital wallet.
```

File: developers/verifiers/index.md [hunk 3]
```diff
@@ -27,13 +29,13 @@ await credentialHandlerPolyfill.loadOnce();
 
 Or, if you're developing on Node.js, add the credential-handler-polyfill library to your project...
 
-```
+```sh
 npm i credential-handler-polyfill@3.0.1
```
Comment (msporny) [inline developers/verifiers/index.md:33]: @davidlehn @dlongley do we really want to specify a polyfill version here?

**Review by msporny — APPROVED** [review]
Minor nit and question, feel free to take the change or leave it.

File: developers/verifiers/index.md (hunk 3 above)
Comment (davidlehn) [reply developers/verifiers/index.md:33]: There are other places with specific versions too.  I'm not sure what best practice is for this without invoking complex tooling. For npm it's probably not too bad and would pick up newer semantic version.  But for the script tags with explicit versions, they will be locked at older version.  If nothing else, could add short notes to check for newer versions and link to the releases page.

File: developers/verifiers/multicredentials.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+The [CHAPI Playground](https://playground.chapi.io/) supports the delivery of
+`QueryByExample` requests for doing Multi-Credential Verification as a
+[Verifiable Credential Verifier](./).
+
+## Wallet Sign Up
+If you already have a CHAPI-enabled digital wallet ([Veres Wallet](https://demo.vereswallet.dev/), [LearnCard](https://learncard.app/), etc.), you can skip to the [VC Issuance section](#vc-issuance).
+
+Here's how to get a demo Veres Wallet if you need one:
+
+1. Visit <https://demo.vereswallet.dev/> and click "Try Veres Wallet"
```
Comment (davidlehn) [inline developers/verifiers/multicredentials.md:17]:
```suggestion
1. Visit <https://demo.vereswallet.dev/> and click "Try Veres Wallet".
```

File: developers/verifiers/multicredentials.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+2. Use an email address that you can access. The Veres Wallet supports email address tags, so you can modify a valid email address by adding "+chapi" to create multiple accounts: myname+chapi@example.com The "+chapi" can be anything you like and you can use this approach to make multiple accounts at a single email address.
+
+3. Once you click "Register", you'll receive an "Allow Wallet" modal. Make sure to click "Allow" on that modal, so the wallet can handle credentials on your behalf.
+
+Once that's done, proceed to VC Issuance.
+
+## VC Issuance
+
+1. Visit the Issuer section of the CHAPI Playground by visiting <https://playground.chapi.io/issuer>
```
Comment (davidlehn) [inline developers/verifiers/multicredentials.md:27]:
```suggestion
1. Visit the Issuer section of the CHAPI Playground by visiting <https://playground.chapi.io/issuer>.
```

File: developers/verifiers/multicredentials.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+5. You'll land at the main CHAPI Playground screen. Click "Issuer Demo".
+
+Next, follow the above process a few more times to load up your wallet with example credentials.
+
+## Multiple VC Verification
+
+Now that you have several VC's in your wallet, let's get them all verified in one request.
+
+1. Visit the playground's Verifier section at <https://playground.chapi.io/verifier>
```
Comment (davidlehn) [inline developers/verifiers/multicredentials.md:43]:
```suggestion
1. Visit the playground's Verifier section at <https://playground.chapi.io/verifier>.
```

File: developers/verifiers/multicredentials.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+2. Click each of the VC's you loaded into your wallet (i.e. "Alumni", "JFF x vc-edu PF2", and "Permanent Resident" for example).
+
+3. Click "Request Verifiable Presentation" to request that your wallet share these credentials.
+
+4. In the demo Veres Wallet you'll see each of the credentials you selected (plus the ability to select which ones you want to verify). Click the "Share" button to send them to the playground for verification.
+
+5. The process should end with a VerifiablePresentation JSON document and the words "Presentation Verified"
```
Comment (davidlehn) [inline developers/verifiers/multicredentials.md:51]:
```suggestion
5. The process should end with a VerifiablePresentation JSON document and the words "Presentation Verified".
```

File: developers/verifiers/index.md (hunk 3 above)
Comment (dlongley) [reply developers/verifiers/index.md:33]: We should probably not specify a version -- provided that we remember to keep these docs up to date when we make any breaking changes.

File: developers/verifiers/multicredentials.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+4. Once you've completed DID Authentication (or not), you'll be provided with the Verifiable Credential JSON. Click "Store in Wallet" to proceed.
+
+5. You'll land at the main CHAPI Playground screen. Click "Issuer Demo".
+
+Next, follow the above process a few more times to load up your wallet with example credentials.
+
+## Multiple VC Verification
+
+Now that you have several VC's in your wallet, let's get them all verified in one request.
```
Comment (dlongley) [inline developers/verifiers/multicredentials.md:41]:
```suggestion
Now that you have several VCs in your wallet, let's get them all verified in one request.
```

**Review by dlongley — APPROVED** [review]
I didn't look closely at this but don't want it to be held up based on my review. RSLGTM.

File: developers/verifiers/multicredentials.md (hunk 8 above)
Comment (BigBlueHat) [reply developers/verifiers/multicredentials.md:41]: Fixing this elsewhere also.

**Pull request merged**